docs/application/dotnet/guides/nui/widget.md

```diff
@@ -0,0 +1,242 @@
+
+The widget instance has its own life-cycle similar to that of the widget application. However, the widget instance is an object created by the widget viewer application. Many widget instances can be running on a widget application process.
+
+> [!NOTE]
+> The case to use many widget instances in one widget application is known as multi-instance. In some devices, the multi-instance may not be supported. If a device does not support multi-instance, an error message is displayed.
+
+[`WidgetView`](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.WidgetView.html) shows the contents drawn by [`Widget`](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Widget.html) on the screen.
+
+To summarize, you [create a NUI WidgetApplication](#create-a-nui-widget-application), [create a NUI WidgetView](#create-a-nui-widget-view), and use Widget to view layout on the screen.
+
+## Instance States and Events
+
+The `Widget` class provides interface for creating custom widget.
+The following table lists the main functions to manage the widget instance:
+
+**Table: Main functions**
+| Function           | Description                                                                                   |
+|--------------------|-----------------------------------------------------------------------------------------------|
+| `OnCreate()`       | This function is called after the widget instance is created.                                 |
+| `OnTerminate()`    | This function is called after the widget instance is terminated.                              |
+| `OnPause()`        | This function is called when the widget is invisible.                                         |
+| `OnResume()`       | This function is called when the widget is visible.                                           |
+| `OnResize()`       | This function is called before the widget size is changed.                                    |
+| `OnUpdate()`       | This function is called when an event for updating the widget is received.                    |
+| `SetContentInfo()` | This function sends data about current state of widget instance to a widget viewer application. If you want to save the current state of widget before deleting it, you can save it to the current status using `SetContentInfo()`.|
+
+Most of functions are used for managing widget instance life-cycle.
+
+## Create a NUI Widget Application
+
+If you want to use NUI for your widget application, you must create the application with [NUI Widget Application](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIWidgetApplication.html) class. 
+
+To create a NUI widget application:
+1. To use the methods and properties of [Tizen.NUI](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.html) namespace, include the following code:
+
+   ```csharp
+   using Tizen.NUI;
+   ```
```

Comment:
@surbhig12 If I miss something, please add it with your comment.

If you want to move this page to guide/applications,

1. please add  "Prerequisites" section as below.

---

## Prerequisites

To enable your application to use the NUI widget functionality:

1. To use the methods and properties of [Tizen.NUI](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.html) namespace, include it in your application:

   ```csharp
   using Tizen.NUI;
   ```

## Create a NUI Widget Application

If you want to use NUI for your widget application, you must create the application with [NUI Widget Application](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIWidgetApplication.html) class. 

1. Define your widget application class, which is inherited from [NUIWidgetApplication](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIWidgetApplication.html) class:

   ```csharp
   class Program : NUIWidgetApplication
   ```
...

---


2. Also, applications/overview.md should be updated to add a link and description about widget.md.
   Enhancing "A widget is used to create light Tizen .NET applications on the home screen." used in /ui-components.md would work.

3. toc_all.md must be updated also to show the new location.

Reply:
@mijin85cho, thank you for pointing this out.

@tscholb,

Request you to please update the page as suggested by @mijin85cho.

Please see other guides for reference: https://docs.tizen.org/application/dotnet/guides/applications/watch-app/